libs/core/src/lib/popover/popover-body/popover-body.component.scss

```diff
@@ -150,6 +151,54 @@ $fd-popover-background-color: #fff;
 			}
 		}
 	}
+
```

Comment:
Please create a mixin:
```
@mixin set-border-radius($position, $radius) {
		border-#{$position}-right-radius: $radius;
    	border-#{$position}-left-radius: $radius;
}
```

Then you can use it like this:
```
@include set-border-radius(top, $fd-popover-border-radius);
```

or even 

```
@mixin set-border-radius($position, $radius: $fd-popover-border-radius) {
		border-#{$position}-right-radius: $radius;
    	border-#{$position}-left-radius: $radius;
}
```

and then:
```
@include set-border-radius(top);
```

You will avoid a looot of code :)